vercel/validator_map_max_count.go

```diff
@@ -0,0 +1,42 @@
+package vercel
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var _ validator.Map = validatorMapMaxCount{}
+
+func mapMaxCount(max int) validatorMapMaxCount {
+	return validatorMapMaxCount{
+		Max: max,
+	}
+}
+
+type validatorMapMaxCount struct {
+	Max int
+}
+
+func (v validatorMapMaxCount) Description(ctx context.Context) string {
+	return fmt.Sprintf("Map must contain fewer than %d items", v.Max)
+}
+func (v validatorMapMaxCount) MarkdownDescription(ctx context.Context) string {
+	return fmt.Sprintf("Map must contain fewer than %d items", v.Max)
+}
```

Comment:
We are doing `req.ConfigValue.Elements()) > v.Max` below, shouldn't it be "can't contain more than %d items"?